TrainingContainer.Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM python:3.9-slim
+
+WORKDIR /
+
+COPY requirements.txt .
+RUN apt-get update -y && apt-get install -y gcc
+RUN pip install -r requirements.txt
+COPY . .
+
```

Comment:
```suggestion

RUN curl "https://awscli.amazonaws.com/awscli-exe-linux-x86_64.zip" -o "awscliv2.zip"
RUN unzip awscliv2.zip
RUN ./aws/install

ARG AWS_REGION
ARG AWS_DEPLOY_ACCESS_KEY_ID
ARG AWS_DEPLOY_SECRET_ACCESS_KEY

RUN aws configure set region $AWS_REGION
RUN aws configure set aws_access_key_id $AWS_DEPLOY_ACCESS_KEY_ID
RUN aws configure set aws_secret_access_key $AWS_DEPLOY_SECRET_ACCESS_KEY
```

Reply:
I'm definitely okay with the x86_64 version of aws cli as long as its linux based

Reply:
fixed @NMBridges

Reply:
see PR into `dev` created today! @farisdurrani